README.md

```diff
@@ -10,6 +10,8 @@
 <a href=https://github.com/ambv/black style="margin-left: 10px">
     <img src="https://img.shields.io/badge/code%20style-black-000000.svg">
 </a>
+
+[![Join the chat at https://gitter.im/prefectio/prefect](https://badges.gitter.im/prefectio/prefect.svg)](https://gitter.im/prefectio/prefect?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
This has to be formatted as an `<a>` and an `<img>` tag, as above, in order to be properly centered in github.

Reply:
👍 updated